ovh/config.go

```diff
@@ -63,15 +65,21 @@ func (c *Config) loadAndValidate() error {
 	}
 
 	if !c.authenticated {
-		var cred OvhAuthCurrentCredential
-		if err := c.OVHClient.Get("/auth/currentCredential", &cred); err != nil {
+		var me MeResponse
+		if err := c.OVHClient.Get("/me", &me); err != nil {
```

Comment:
Change here brings 2 problems:
- `GET /me` requires a specific right on the IAM, comparing to `/auth/` routes.
- Canadian IS returns the email address instead of the nichandle on `GET /me`, that would be incompatible with IAM URN.

We should do a change on backend side on route `GET /auth/details` to return the nichandle, and then use it instead of.